pyrpl/widgets/module_widgets/scope_widget.py

```diff
@@ -121,7 +121,7 @@ def init_gui(self):
 
         #self.setLayout(self.main_layout)
         self.setWindowTitle("Scope")
-        self.win = pg.GraphicsWindow(title="Scope")
+        self.win = pg.GraphicsLayoutWidget(title="Scope")
```

Comment:
On this change I also add "show=True" so that the widget gets shown on creation.